statistics/EzySimpleComponentStatistics.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace com.tvd12.ezyfoxserver.client.statistics
+{
+    public class EzySimpleComponentStatistics : EzyComponentStatistics
+    {
+        protected EzyNetworkStats networkStats;
+
+        public EzyNetworkStats getNetworkStats()
+        {
+            return this.networkStats;
+        }
+
+        protected EzyNetworkStats newNetworkStats()
+        {
+            return new EzySimpleNetworkStats();
+        }
+
+        public EzySimpleComponentStatistics()
+        {
+            networkStats = newNetworkStats();
```

Comment:
ở trên đã có `this.` thì ở dưới cũng nên có `this.` cho đồng bộ em ạ

Reply:
em đã fix